### DISCUSSION TITLE
How to deactivate/revoke a machine to make the license available for activating another machine?

### DISCUSSION THREAD

**Discussion by sesamzoo** [cymian/mouseless#66] [Q&A] [answered]
According to https://mouseless.click/terms/:
> A single license permits program activation on up to two (2) machines at a time.

How could I deactivate a machine so that I can migrate to another machine and use my license there, f. i. when I replace my machine with a new one?

More specific use case/question: I'll have to reset one of my two machines in the next couple of weeks. How can I make sure that I can still use my license after setting up the machine again?

---

I am really looking forward to using your app. Great idea!

**Comment by cymian** [comment, marked as answer]
Hi @sesamzoo, great question!

When you start up the app on a third machine, you will get a dialog asking which of the prior machines (labeled by date/time added) you would like to deactivate.

As long as there isn't clear signs of abuse, a user should have no issues with this activation/deactivation process.  You should be totally fine in this case.  Please do let me know if you have any issues with the process!

I will update the documentation to include this info :)

Really glad to hear you're excited about mouseless! Thank you for the question!

**Reply by sesamzoo** [reply]
That's good to know, thank you for the quick response, @croian.
